error.html

```diff
@@ -1,23 +0,0 @@
----
```

Comment:
This page was served for the `5xx` error pages encountered when serving the website. Because GitHub is now taking care of that, there's no `5xx` status codes for us to deal with, so the page is redundant.